drools-wb-services/drools-wb-verifier/drools-wb-verifier-backend/src/test/java/org/drools/workbench/services/verifier/webworker/client/testutil/TestUtil.java

```diff
@@ -29,123 +30,130 @@
 
 public class TestUtil {
 
-    public static String loadResource( final String name ) throws
-                                                           Exception {
-        final InputStream in = DecisionTableAnalyzerFromFileTest.class.getResourceAsStream( name );
-        final Reader reader = new InputStreamReader( in );
+    public static String loadResource(final String name) throws
+            Exception {
+        final InputStream in = DecisionTableAnalyzerFromFileTest.class.getResourceAsStream(name);
+        final Reader reader = new InputStreamReader(in);
         final StringBuilder text = new StringBuilder();
         final char[] buf = new char[1024];
         int len = 0;
-        while ( ( len = reader.read( buf ) ) >= 0 ) {
-            text.append( buf,
-                         0,
-                         len );
+        while ((len = reader.read(buf)) >= 0) {
+            text.append(buf,
+                        0,
+                        len);
         }
         return text.toString();
     }
 
-    public static void assertOnlyContains( final Set<Issue> result,
-                                           final String... expected ) {
+    public static void assertOnlyContains(final Set<Issue> result,
```

Comment:
It seems that we could replace these methods by an AssertJ call. (Perhaps first converting the `Issue`s to `String`s using a very simple, clean stream.)